src/components/Shell/Shell.tsx

```diff
@@ -234,6 +259,12 @@ export const Shell = ({ appNeedsUpdate, children, userPeerId }: ShellProps) => {
             isOpen={isQRCodeDialogOpen}
             handleClose={handleQRCodeDialogClose}
           />
+          <RoomShareDialog
+            roomId={roomId ?? ''}
```

Comment:
I like having roomId as a nullable type, because there isn't always a room, and we don't want client code having to validate the roomId string (checking it's not empty). I feel it is more explicit checking for null or undefined.